x-pack/plugins/cases/server/features.ts

```diff
@@ -57,6 +68,9 @@ export const getCasesKibanaFeature = (): KibanaFeatureConfig => {
           read: [...filesSavedObjectTypes],
         },
         ui: capabilities.all,
+        replacedBy: [
+          { feature: FEATURE_ID_V2, privileges: ['minimal_all', 'create_comment', 'case_reopen'] },
```

Comment:
@azasypkin Why do we have to define it like this and not like `privileges: ['all']`

Reply:
It actually depends on your requirements and whether you want roles with the legacy `all` and `read` privileges to automatically receive any future sub-feature privileges that you might introduce with the `includeIn: 'all' | 'read'` property, or if you want them to only retain the privileges they had before the new feature was introduced, limiting any future expansion of their scope. I assume in your case, you do want deprecated `all/read` to "follow" the new `all/read`, so your config should look like this instead:
```ts
replacedBy: {
  default: [{ feature: FEATURE_ID_V2, privileges: ['all']],
  minimal: [
    { feature: FEATURE_ID_V2, privileges: ['minimal_all', 'create_comment', 'case_reopen'] },
  ]
}
```

Reply:
@cnasikas - I referenced the documentation I found about this here: https://github.com/elastic/kibana/pull/194898#discussion_r1800339848

Also updated one of `cases_delete` roles I found in the api tests as that currently is set as `['read', 'cases_delete']` vs `['minimal_read', 'cases_delete']` which I don't think is valid, but maybe I'm wrong there..